uncompyle6/semantics/pysource.py

```diff
@@ -661,21 +661,33 @@ def n_ifelsestmt(self, node, preprocess=False):
           if ..
           elif ...
 
+          [else ...]
+
         where appropriate
         """
         else_suite = node[3]
 
         n = else_suite[0]
+        old_stmts = None
 
         if len(n) == 1 == len(n[0]) and n[0] == 'stmt':
             n = n[0][0]
         elif n[0].kind in ('lastc_stmt', 'lastl_stmt'):
             n = n[0]
-            if n[0].kind in ('ifstmt', 'iflaststmt', 'ifelsestmtl'):
+            if n[0].kind in ('ifstmt', 'iflaststmt', 'iflaststmtl', 'ifelsestmtl', 'ifelsestmtc'):
                 # This seems needed for Python 2.5-2.7
                 n = n[0]
                 pass
             pass
+        elif ( len(n) > 1 and 1 == len(n[0]) and n[0] == 'stmt'
```

Comment:
I am not sure about this stuff. It was part of my trying to merge diffs. I think you said this was no longer relevant?